srcpkgs/openspades/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'openspades'
+# The OpenSpades-Non-GPL-Pak License states that the pak000-Nonfree.pak
+# has to be redistributed together with the binary, so never split it
+# into a -data subpackage !!!
+pkgname=openspades
+version=0.1.2
+revision=1
+create_wrksrc=true
+build_wrksrc="openspades-${version}"
+build_style=cmake
+configure_args="-DOPENSPADES_INSTALL_BINARY=bin -DOPENSPADES_RESOURCES=NO"
+hostmakedepends="pkg-config unzip zip"
+makedepends="glew-devel libcurl-devel SDL2-devel SDL2_image-devel
+ freealut-devel freetype-devel opus-devel opusfile-devel"
+depends="xdg-utils"
```

Comment:
`depends="libopenal xdg-utils"`

Because:
````
Failed to load a OpenAL driver.
libopenal.so.1:
Failed to dlload 'libopenal.so.1': Error loading shared library /usr/bin//libopenal.so.1: No such file or directory
````

Reply:
fixed it. Thanks for testing.

Reply:
I prefer to be cautious. :stuck_out_tongue_winking_eye: